osu.Framework/Testing/TestScene.cs

```diff
@@ -69,7 +112,7 @@ public void SetupGameHost()
         [OneTimeTearDown]
         public void DestroyGameHost()
         {
-            host.Exit();
+            (host as TestSceneHost)?.ExitFromRunner();
```

Comment:
nit: would make sense to direct cast since `DestroyGameHost` should only be invoked in headless context as far as I'm aware.
```suggestion
            var headlessHost = (TestSceneHost)host;
            headlessHost.ExitFromRunner();
```

Reply:
I was doing it for the null check, which I think we want.

Reply:
`host` was previously accessed without a null conditional though?

Reply:
I guess you could say that, yeah.